atd-vze/src/queries/recommendations.js

```diff
@@ -15,23 +15,9 @@ export const GET_RECOMMENDATION_LOOKUPS = gql`
 
 export const INSERT_RECOMMENDATION = gql`
   mutation InsertRecommendation(
-    $text: String
```

Comment:
mutation was failing silently if you try to create a recommendation by first adding recommendation text because this mutation was expecting the variable `text` but we were feeding it `rec_text`